### ISSUE TITLE
DB version 5.0.0-alpha-dirty - 'std::out_of_range'

### ISSUE THREAD

**Issue by TrevorNBudd** [APN-Pucky/tyrant_optimize#48]
I get a `DB version 5.0.0-alpha-dirty` - `'std::out_of_range'` error when I try to run TUO. This is with a fresh install (i.e. just unzipped TUO v6.2.0 and tried to run it). Both TUO and TUO-DEBUG affected.
```
TUO v6.2.0> .\tuo.exe "Tabitha" "Cyrus" +v
DB version 5.0.0-alpha-dirty
.\tuo.exe : terminate called after throwing an instance of 'std::out_of_range'
At line:1 char:1
+ .\tuo.exe "Tabitha" "Cyrus" +v
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (terminate calle...::out_of_range':String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
  what():  basic_string::substr: __pos (which is 138) > this->size() (which is 137)

TUO v6.2.0> .\tuo-debug.exe "Tabitha" "Cyrus" +v
DB version 5.0.0-alpha-dirty
.\tuo-debug.exe : terminate called after throwing an instance of 'std::out_of_range'
At line:1 char:1
+ .\tuo-debug.exe "Tabitha" "Cyrus" +v
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (terminate calle...::out_of_range':String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
  what():  basic_string::substr: __pos (which is 138) > this->size() (which is 137)
```

**Comment by TrevorNBudd** [comment]
I removed "database.yml" (i.e. renamed it to "database.yml.bak") then tried TUO again with no errors. When I tried it with operation "climb", the database was recreated with no errors.

**Comment by TrevorNBudd** [comment]
Actually. It appeared v6.2.0 shipped with the v5.0.0 database, which was incompatible. Line 1 in the shipped database says `version: 5.0.0-alpha-dirty`. I think Line 1 in the recreated database said `version: v6.2.0-debug` (I already deleted that version).

Going forward with newer versions, it might be better to ship them without the database and let TUO create the correct database on first use.

I think you can close this issue now.
